library/Makefile

```diff
@@ -63,6 +63,8 @@ DLEXT = dylib
 endif
 endif
 
+VPATH = ../tinycrypt
```

Comment:
Oh, that's nice, I didn't know about `VPATH` before. I always enjoy learning new tricks while reviewing PRs :)